Source/ACE/Entity/Creature.cs

```diff
@@ -495,5 +497,162 @@ protected virtual void UpdateVitalInternal(CreatureVital vital, uint newVal)
                 new ActionChain(this, () => VitalTickInternal(vital)).EnqueueChain();
             }
         }
+
+        public override void SerializeIdentifyObjectResponse(BinaryWriter writer)
```

Comment:
in short, the way we key object creation is bad and unsustainable. ObjectDescriptionFlags is more changeable than we originally suspected.